bin/dquota.py

```diff
@@ -140,12 +142,13 @@ def main():
                 stats["%s_users" % (storage_name,)] = 0
                 logger.debug("storage_name %s found no users who are exceeding their quota" % storage_name)
 
-    except Exception, err:
+    except Exception as err:
         logger.exception("critical exception caught: %s" % (err))
         opts.critical("Script failed in a horrible way")
         sys.exit(NAGIOS_EXIT_CRITICAL)
```

Comment:
so this is wrong, should exit with `NAGIOS_EXIT_CRITICAL[0]` (and all pplaces where this is called)

Reply:
I can change it but any other script I look at uses the same line?

Reply:
it is not even needed. the opts.critical on the line above it will exit for you.